apps/condo/domains/property/components/panels/Builder/MapConstructor.spec.ts

```diff
@@ -227,16 +247,16 @@ describe('Map constructor', () => {
         describe('Check that JSON schema validator is working', () => {
             it('should react to bad structure', () => {
                 const Building = new MapEdit(null, () => null )
-                Building.map = cloneDeep(notValidBuildingMapJson)
+                Building.map = cloneDeep(notValidBuildingMapJson) as BuildingMap
```

Comment:
why typecast?

Reply:
TS issues warning after we change type from JSON to Apollo schema